### ISSUE TITLE
Error: Unexpected token =

### ISSUE THREAD

**Issue by jlow-mudbath** [acacode/swagger-typescript-api#136]
Running `npx swagger-typescript-api -p ./src/openapi.yml -o ./src -n types.ts --no-client`
shows the following error:
`Unexpected token =`
when using version 4.0.5

If I use version 4.0.4 (`swagger-typescript-api@4.0.4`) then there are no errors and it runs successfully

Note: there are no `=` symbols in the yaml file.

**Comment by js2me** [comment]
Hello @jlow-mudbath ! Can you share full log of ther error ?

**Comment by jlow-mudbath** [comment]
This is the full output I get:

```
PS C:\Repos\__redacted__> npx swagger-typescript-api -p ./src/openapi.yml -o ./src -n types.ts --no-client
npx: installed 53 in 4.284s
Unexpected token =
```

Windows 10
Powershell 7.1.1
node 10.16.3

**Comment by js2me** [comment]
@jlow-mudbath that's interesting problem, can you provide swagger schema ?

**Comment by js2me** [comment]
@jlow-mudbath Sorry, I don't needed swagger schema, I guess I know what's problem   
This is old version of nodejs which is not support this syntax  
```diff
class Foo {
+   func = () => {}
}
```

**Comment by js2me** [comment]
@jlow-mudbath Will fix that at next release

**Comment by jlow-mudbath** [comment]
Thanks @js2me

**Comment by js2me** [comment]
Good day @jlow-mudbath !   
This problem should be fixed in 4.1.0.   
If you still get this problem, please reopen issue!
